recipes/nlohmann_json/all/conanfile.py

```diff
@@ -50,4 +49,4 @@ def package(self):
 
 
     def package_id(self):
-        self.info.header_only()
+        self.info.settings.clear()
```

Comment:
so, it's not a header-only library?

Reply:
It's, but the option `multiple_headers` changes the number of headers in the package. The `header_only()` provides the same PKG ID even for different options.

Actually, we don't need `package_id` anymore, since there is no settings in the recipe.